src/Card.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+
+function Card (props) {
+    return (
+        <div className="card">
+            <div className="left-column">
+            <img className="thumbnail" src={props.img} alt="avatar.img"/>
```

Comment:
alt should be from props

Reply:
could you, please, explain this one more? how alt should be from props? also if I name it better, I should name each picture separately then?

Reply:
Teoriškai jei nuotraukoje yra šuo, tai rašai alt="dog", ne "dog.img", plius pas tave visos nuotraukos turi dabar tą patį alt.

Reply:
tai kadangi kiekviena nuotrauka yra unikali, laikyk json'e tekstą papildomą "imageName", kurį naudosi kaip "alt" tekstą.